uat/testing-features/src/main/java/com/aws/greengrass/steps/MqttControlSteps.java

```diff
@@ -0,0 +1,39 @@
+package com.aws.greengrass.steps;
+
+import io.cucumber.guice.ScenarioScoped;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.When;
+
+@ScenarioScoped
+public class MqttControlSteps {
+
+    @And("the control component configured:")
```

Comment:
FYI: Control is not a component of greengrass.

Reply:
I see. But the word "component" is also a common name for classes injected using DI frameworks (for example, in Dagger from OTF). Could you suggest how to rename this step to avoid misunderstanding?

Reply:
1) "MQTT Client Control is configured:"
2) Remove configuration - port etc.

Reply:
fixed 7bec4aab4bf10f182d29bef87204a704dac3bdce